src/components/GroupOccupancy/Units.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import solidMale from '../../assets/occupancy-icons/solid-male.svg'
+import outlineMale from '../../assets/occupancy-icons/outline-male.svg'
+
+import classes from './styles.module.scss'
+
+const _ = require('lodash')
+const MAX_FIGURES = 10
+
+const Units = ({ groupCapacity, occupancyPercentage }) => {
+  const amountOfPersonIcons = (groupCapacity < MAX_FIGURES) ? groupCapacity : MAX_FIGURES
+  const percentageNumber = parseInt(String(occupancyPercentage).replace('%', ''))
```

Comment:
I'd be careful with this since it relies knowing that the percentage is formatted in a certain way,but it's ok for now, do not change it!